schema/collection.js

```diff
@@ -59,6 +59,19 @@ const CollectionType = new GraphQLObjectType({
   },
 })
 
+// This resolver is re-used by `me { saved_artworks }`
```

Comment:
The root one can be used for any collection of Artworks, this is a specific one that we use for saved_artworks